ouroboros-consensus-cardano/changelog.d/20240111_195320_facundo.dominguez_lightweight_checkpointing.md

```diff
@@ -0,0 +1,24 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+<!--
+### Non-Breaking
+
+- A bullet item for the Non-Breaking category.
+
+-->
+### Breaking
+
+- Implement lightweigth checkpointing [#449](https://github.com/IntersectMBO/ouroboros-consensus/issues/449).
+  A validation to help Genesis nodes follow the historical chain. A new field `cardanoCheckpoints`
```

Comment:
I wouldn't mention Genesis here, it applies just as well to Praos nodes (indeed, checkpoints could be seen as an *alternative* to Genesis, though we only intend to use complementarily where using Genesis is either somewhat subtle (Byron) or just doesn't work (very big hypothetical mainnet outage).